### PR TITLE
DX-766 feat: add `allowed admins` prompt into application init flow

### PR DESCRIPTION
**Pull request by Fadhil**
## Summary
We add a prompt for the user to enter `allowed admins` (as a list of emails), to limit the ability of launching a liveview instance to these admins only.

Screenshot 2024-08-20 at 5 10 52 PM

Screenshot 2024-08-20 at 5 11 12 PM

Screenshot 2024-08-20 at 5 11 25 PM

An actual update to `wukong-ci-mock`'s `.wukong.toml` showed the config reflected in `ApplicationConfig` on Wukong prod.

Screenshot 2024-08-21 at 1 17 19 PM

Ticket: https://mindvalley.atlassian.net/browse/DX-766

## What's Changed

Prompts for `allowed admins` in `application init` flow.

### CODE REVIEW

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/226#" title="Author self-approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/226#pullrequestreview-2259639171" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
